checkov/terraform/graph_builder/local_graph.py

```diff
@@ -176,6 +139,21 @@ def get_module_vertices_mapping(self) -> None:
         For each vertex, if it's originated in a module import, add to the vertex the index of the
         matching module vertex as 'source_module'
         """
+        if strtobool(os.getenv('CHECKOV_NEW_TF_PARSER', 'False')):
+            for vertex in self.vertices:
+                if not vertex.source_module_object:
+                    continue
+                for i, source_v in enumerate(self.vertices):
```

Comment:
what about the foreach index? @ChanochShayner

Reply:
We are not supporting the foreach index in modules for now.
When we will support Ill add it